### ISSUE TITLE
FiveM Integrations code 100

### ISSUE THREAD

**Issue by IlyaMix01** [SnailyCAD/snaily-cadv4#924]
### Describe the feature

Create an opportunity so that when the code 100 is turned on, this is also displayed on the server

**Comment by jxckUK** [comment]
You can do this using the public API on `https://api.projecthyde.com/v1/admin/manage/cad-settings`.

It is under `miscCadSettings.signal100Enabled`

**Comment by Dev-CasperTheGhost** [comment]
Yep, what jack said. There are many things you can do with the API
